### ISSUE TITLE
Feature Request: Improve Lighthouse Best Practices Score

### ISSUE THREAD

**Issue by wordshaker** [DDDEastMidlandsLimited/dddem-web#214]
### Describe the feature

The site currently has a score of 86 for Best Practices when Lighthouse is run against it. 

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110367850-1f84ef00-8040-11eb-8529-92747daf389e.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the Best Practices of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse score for Best Practices is increased to at least 90.
